presto-mysql/src/test/java/io/prestosql/plugin/mysql/TestingMySqlServer.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.mysql;
+
+import org.testcontainers.containers.MySQLContainer;
+
+import java.io.Closeable;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+import static java.lang.String.format;
+import static org.testcontainers.containers.MySQLContainer.MYSQL_PORT;
+
+public class TestingMySqlServer
```

Comment:
Consider making this extend `MySQLContainer` like I did in 79b5d4f064ab934d6a53ea0972cf821a932db1b2. That avoids needing all of the pass-through methods.